emcc.py

```diff
@@ -1339,10 +1339,10 @@ def has_c_source(args):
     if shared.Settings.USE_WEBGL2:
       shared.Settings.MAX_WEBGL_VERSION = 2
 
-    forced_stdlibs = []
+    include_stdlibs = []
```

Comment:
`included`, with a `d` perhaps? at least for me, `include_stdlibs` sounds like a boolean, not a list of included stdlibs.